### ISSUE TITLE
[BUG] SKSEVR for SkyrimVR does not get correct controller info for Index Knuckles

### ISSUE THREAD

**Issue by holon-holon** [ValveSoftware/SteamVR-for-Linux#718]
**Describe the bug**
SKSEVR fails to retrieve meaningful controller info from OpenVR, and defaults back to Vive Wands. Manifests as menus being broken due to control sticks being detected as touch inputs, and VRIK and HIGGS being broken due to detecting Vive Wands.

**To Reproduce**
1. Start SkyrimVR via the SKSE loader
2. Observe that menus and controls are broken

**Expected behavior**
SKSEVR correctly detects the Knuckles and the controls and mods work properly.

**System Information (please complete the following information):**
* Steam client version (build number or date): 1719618357
* SteamVR version: beta
* Distribution (e.g. Ubuntu): Fedora 40
* Steam runtime diagnostics: [steam_runtime_diag.txt](https://github.com/user-attachments/files/16042420/steam_runtime_diag.txt)
* Steam and SteamVR logs: [steam-logs.tar.gz](https://github.com/user-attachments/files/16042418/steam-logs.tar.gz)
* Minidumps: [steam-minidumps.tar.gz](https://github.com/user-attachments/files/16042417/steam-minidumps.tar.gz)

**Screenshots**

**Additional context**
The game renders the knuckles correctly before the player model loads in the pre-game area.

**Comment by ThaSwapMeetPimp** [comment]
You have the VRIK Bindings correct?

**Comment by holon-holon** [comment]
Yes, that's correct. The bug is in the way SKSEVR tries to get information from OpenVR, as far as I can tell.

**Comment by ThaSwapMeetPimp** [comment]
Hmm....well, I use VRIK and Higgs, I have an Index, and the knuckles shows as Vive Wands during loading, but it works fine. Maybe the modified OpenVR bindings file thats needed for FRIK is necessary for VRIK now also?

**Comment by ThaSwapMeetPimp** [comment]
Yeah, reading your issue and the issue with Fallout that requires a modified OpenVR_api.dll, I would think it is the same issue. Can you go into your controller bindings and tell it to treat your controllers as Oculus Touch controllers?

Edit: Here's a link to the Steam discussions post https://steamcommunity.com/app/611660/discussions/0/1644304412663862614/
